src/Microsoft.DotNet.Build.Tasks/GenerateEncodingTable.cs

```diff
@@ -0,0 +1,395 @@
+﻿using Microsoft.Build.Framework;
```

Comment:
Do you mean every time somebody submits a new PR (during test builds)?  As a task, it only rebuilds the output file when the input file(s) changes.  This is what happens with all the resource generation, too, which probably don't change that much more.